lib/id3c/db/__init__.py

```diff
@@ -261,6 +287,9 @@ def upsert_sample(db: DatabaseSession,
         if collection_identifier and sample.collection_identifier and collection_identifier != sample.collection_identifier:
             LOG.warning(f"upsert_sample: collection_identifier is changing on sample {sample.id} from {sample.collection_identifier} to {collection_identifier}")
 
+        if sample.identifiers_changed is None and update_identifiers:
```

Comment:
If the change suggested above is made, this should be able to be changed to check for `False` instead of `None`.